modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
@wing328 idea is to remove the HTTP client implementation completely and inject a [PSR-7](http://www.php-fig.org/psr/psr-7/) capable client such as [Guzzle](https://github.com/guzzle/psr7).

This means setting up these kinds of configuration would be outside of scope of generated code, it would assume the injected client is capable of being configured properly in its own right. This code would just use it as-is.